### DISCUSSION TITLE
Im new to this SFP module world and i concerned about the module temperture...

### DISCUSSION THREAD

**Discussion by kingsvell** [Anime4000/RTL960x#253] [Q&A] [answered]
![image](https://github.com/Anime4000/RTL960x/assets/39862389/78944edb-08ce-4a4c-940f-aabb017b838c)

The temperature showed in the status page is related to the point of the fiber or the module? The module is pretty hot, and im worried to burn out the module. 

Im using the firmware version: V1.7.1-220916 

Ive transformed an old switch HP v1910 into a router using this module and this firmware.

**Comment by Anime4000** [comment, marked as answer]
The temperature is typical can handle max as 80°c

the temperature is shared between SoC and BOSA (Laser)

my temperature of PON Stick run at 68°c all time since then, no issue so far

**Reply by kingsvell** [reply]
Thanks, Anime!
